src/bundle/Resources/views/themes/admin/ui/component/alert/alert.html.twig

```diff
@@ -0,0 +1,59 @@
+{% import "@ezdesign/ui/component/macros.html.twig" as html %}
+
+{% set icon_name_map = {
+    info: 'system-information',
+    error: 'circle-close',
+    warning: 'warning-triangle',
+    success: 'checkbox',
+} %}
+
+<div
+    class="alert ibexa-alert ibexa-alert--{{ type }}
+        {{ class|default('') -}}"
+    role="alert"
+    {{ html.attributes(attr|default({})) }}
+>
+    {% block icon %}
+        {% set icon = icon_name_map[type] %}
+        <svg class="ibexa-icon ibexa-icon--small ibexa-alert__icon">
+            <use xlink:href="{{ icon_path is defined ? icon_path : ibexa_icon_path(icon) }}"></use>
+        </svg>
+    {% endblock %}
+    {% block content %}
+        <div
+            class="ibexa-alert__content
+                {% if show_subtitle_beside|default(false) %} ibexa-alert__content--subtitle-below{% endif %}"
+        >
+            {% block title %}
+                {% if title is defined %}
+                    <div class="ibexa-alert__title">
+                        {{ title }}
+                    </div>
+                {% endif %}
+            {% endblock %}
+            {% block subtitle %}
+                {% if subtitle is defined %}
+                    <div class="ibexa-alert__subtitle">
+                        {{ subtitle }}
+                    </div>
+                {% endif %}
+            {% endblock %}
+            {% block custom_wrapper %}
+                <div class="ibexa-alert__custom">
```

Comment:
`custom` is not meaningful. What is custom?

Reply:
It's custom stuff, which someone may want to add, e.g. some button. What name would you like to use here?

Reply:
extra_content, additional_content?

Reply:
Changed to `extra_content`.